lib/vtls/vtls.c

```diff
@@ -505,7 +505,6 @@ CURLcode Curl_ssl_addsessionid(struct Curl_easy *data,
     conn->host.name;
 #else
   /* proxy support disabled */
```

Comment:
The comment is probably misplaced as well

Reply:
Not necessarily, but I also don't think it helps anyone...